packages/vscode-extension/src/commands/ManageAccounts.ts

```diff
@@ -319,7 +321,7 @@ export default class ManageAccounts extends BaseAction {
 		};
 		if (url) {
 			commandParams.url = url;
-			commandParams.dev = true;
+			commandParams.dev = url !== ApplicationConstants.PROD_ENVIRONMENT_ADDRESS;
```

Comment:
Do we also need to check account specific url here?

Reply:
I rolled back to the original code here, since I saw there was no difference in behavior.